Station/pycpt-seasonal-station-grid-Chile.ipynb

```diff
@@ -0,0 +1,794 @@
+    "    #   The day field is only used in subseasonal forecasts, otherwise ignored.\n",
+    "    #   The month field is an integer representing a month - ie, May=5.\n",
+    "    'fdate':  dt.datetime(2023, 5, 1),\n",
+    "\n",
+    "    # 'first_year':\n",
+    "    #   The first year of hindcasts you want. **NOT ALL MODELS HAVE ALL YEARS**.\n",
+    "    #   Double check that your model has hindcast data for all years in [first_year, final_year].\n",
+    "    #   This field is defined by a python integer representing a year, e.g. 1993.\n",
+    "    'first_year': 1981,  \n",
+    "\n",
+    "    # 'final_year':\n",
+    "    #   The final year of hindcasts you want. **NOT ALL MODELS HAVE ALL YEARS**.\n",
+    "    #   Double check that your model has hindcast data for all years in [first_year, final_year].\n",
+    "    #   This field is defined by a python integer representing a year, e.g. 2016.\n",
+    "    'final_year': 2021,  \n",
+    "\n",
+    "    # 'target': \n",
+    "    #   Mmm-Mmm indicating the months included in the target period of the forecast. \n",
+    "    #   This field is defined by a python string, with two three-letter month name abbreviations \n",
+    "    #   whose first letters are capitalized, and all other letters are lowercase\n",
+    "    #   and who are separated by a dash character. \n",
+    "    #   For example, if you wanted a JJA target period, you would use 'Jun-Aug'.\n",
+    "    'target': 'Jun',\n",
+    "\n",
+    "    # 'predictor_extent':\n",
+    "    #   The geographic bounding box of the climate model data you want to download.\n",
+    "    #   This field is defined by a python dictionary with the keys \"north\", \"south\",\n",
+    "    #   \"east\", and \"west\", each of which maps to a python integer representing the \n",
+    "    #   edge of a bounding box. i.e., \"north\" will be the northernmost boundary,\n",
+    "    #   \"south\" the southernmost boundary.\n",
+    "    #   Example: {\"north\": 90, \"south\": -90, \"east\": 0, \"west\": 180}\n",
+    "    'predictor_extent': {\n",
+    "        'west': -80, \n",
+    "        'east':  -62,\n",
+    "        'south': -60, \n",
+    "        'north': -12,\n",
+    "      },\n",
+    "\n",
+    "    # 'predictand_extent':\n",
+    "    #   The geographic bounding box of the observation data you want to download.\n",
+    "    #   This field is defined by a python dictionary with the keys \"north\", \"south\",\n",
+    "    #   \"east\", and \"west\", each of which maps to a python integer representing the \n",
+    "    #   edge of a bounding box. i.e., \"north\" will be the northernmost boundary,\n",
+    "    #   \"south\" the southernmost boundary.\n",
+    "    #   Example: {\"north\": 90, \"south\": -90, \"east\": 0, \"west\": 180}\n",
+    "    'predictand_extent': {\n",
+    "        'west':  -80,\n",
+    "        'east': -62,\n",
+    "        'south': -60,\n",
+    "        'north': -12,\n",
+    "      },\n",
```

Comment:
Are you sure you want predictand_extent to be the same as predictor_extent? In the other notebook you made it smaller.

Reply:
This was a quick test to observe the behavior across the region, but in fact, DMC uses several configurations (including this one). The one that usually changes is the predictor. We still haven't defined what the final configuration will be that we'll use to produce the skill atlas, but once we have it, I’ll update this section so that we have this notebook as the 'final operational' product. However, that might happen during the training.